tests/store/createStore.spec.js

```diff
@@ -3,27 +3,23 @@ import {
 } from 'store/createStore'
 
 describe('(Store) createStore', () => {
-  let store
-
-  before(() => {
-    store = createStore()
-  })
-
```

Comment:
reassigning variables in `beforeEach` is a pretty dangerous pattern. You can forget to reassign it and the value will leak into other tests in this file